assets/js/modules/tagmanager/hooks/useExistingTagEffect.test.js

```diff
@@ -70,5 +68,7 @@ describe( 'useExistingTagEffect', () => {
 
 		expect( registry.select( STORE_NAME ).getContainerID() ).toBe( existingContainer.publicId );
 		expect( registry.select( STORE_NAME ).getInternalContainerID() ).toBe( existingContainer.containerId );
+		// Can't seem to prevent an error from updates happening outside of `act()` just for this part.
+		expect( console ).toHaveErrored();
```

Comment:
I think this was from me previously but it would be good to see if this is indeed still needed and see if we can remove the need for it (not worth spending a ton of time on though).

Reply:
This was fixed by importing `actHook` rather than `act` as `renderHook` is being used rather than `render`.